test/decoders/test_video_decoder.py

```diff
@@ -409,6 +416,17 @@ def test_get_frames_at_fails(self, device):
         with pytest.raises(RuntimeError, match="Expected a value of type"):
             decoder.get_frames_at([0.3])
 
+    def test_get_frame_at_av1(self):
+        # We don't parametrize with CUDA because the current GPUs on CI do not
+        # support AV1:
+        decoder = VideoDecoder(AV1_VIDEO.path, device="cpu")
+        ref_frame11 = AV1_VIDEO.get_frame_data_by_index(10)
```

Comment:
The convention we're following in these tests is that the variable name number matches the index number, so this should be `ref_frame10`, `ref_frame_info10` and `decoded_frame10`.